.github/workflows/test.yml

```diff
@@ -1105,8 +1109,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-
-      - uses: ./.github/actions/setup-turborepo-environment
+      - name: Build turborepo CLI from source
```

Comment:
Does `setup-turborepo-environment` build it from source? I thought it just set up the environment

Reply:
It does now, I added it in https://github.com/vercel/turbo/pull/4647